frontend/src/metabase/browse/components/BrowseDatabases.tsx

```diff
@@ -23,7 +26,22 @@ export const BrowseDatabases = ({
 }: {
   databasesResult: ReturnType<typeof useDatabaseListQuery>;
 }) => {
+  const dispatch = useDispatch();
+
   const { data: databases = [], error, isLoading } = databasesResult;
+
+  useEffect(() => {
+    if (error || isLoading) {
+      return;
+    }
+    dispatch(
+      updateSetting({
+        key: "default-browse-tab",
+        value: "databases",
+      }),
+    );
+  }, [error, isLoading, dispatch]);
```

Comment:
ultimately a product call, but I think i'd prefer to put this sort of setting in a localStorage, I think it's fine if it's browser-specific.  I don't like making these API calls every time we switch tabs.